lib/core/utils/preload-cssom.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Returns a CSSStyleSheet object for a given style sheet content/ text
+ * @param {string} cssText content for the stylesheet
+ * @return {CSSStyleSheet}
+ * @private
+ */
+const getCssSheet = (cssText) => {
+	let htmlHead = document.implementation.createHTMLDocument().head;
```

Comment:
We should check the performance cost of creating a new document for every sheet. Perhaps make this so we do this only once per run?